src/webgpu/idl/render_pass/storeOp.spec.ts

```diff
@@ -0,0 +1,14 @@
+export const description = `IDL Tests for RenderPass StoreOp.
+
+  Test Coverage Needed:
+
+  - Test that a color attachment's storeOp is 'store' by default.
+
+  - Test that depthReadOnly is 'false' by default.
+
+  - Test that stencilReadOnly is 'false' by default.`;
```

Comment:
Will move to operation tests with https://github.com/gpuweb/cts/pull/227#discussion_r450854447

Reply:
Done. Removed the entire/idl/ file.